src/main/java/core/basesyntax/service/FileWriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriterService {
+    void writeToFile(String report, String pathToFile);
```

Comment:
```suggestion
public interface FileWriterService {
    void writeToFile(String text, String pathToFile);
```
not critical, but `FileWriterService` shouldn't know anything about reports